.github/actions/poetry/action.yml

```diff
@@ -0,0 +1,89 @@
+name: 'Poetry'
```

Comment:
Why duplicate this action, instead of just referencing the upstream one in Circulation?

Reply:
Good point! Fixed.